Source/CheckoutAPIClient.swift

```diff
@@ -26,15 +26,15 @@ public class CheckoutAPIClient {
 
     init(publicKey: String,
          environment: Environment,
-         correlationIDGenerator: CorrelationIDGenerating,
+         correlationIDGenerator: CorrelationIDManaging,
          logger: FramesEventLogging,
          mainDispatcher: Dispatching,
          networkFlowLoggerProvider: NetworkFlowLoggerProviding,
          requestExecutor: RequestExecuting) {
 
         self.publicKey = publicKey
         self.environment = environment
-        self.correlationIDGenerator = correlationIDGenerator
+        self.correlationIDManager = correlationIDGenerator
```

Comment:
can we change the name of the argument from `correlationIDGenerator` to `correlationIDManager` as well?

Reply:
Makes sense. Will do.